src/data/query-suggestions.json

```diff
@@ -120,11 +123,10 @@
           "404": {
             "$ref": "#/components/responses/IndexNotFound"
           }
-        },
-        "tags": ["query-suggestions"]
+        }
       },
       "put": {
-        "operationId": "customQuerySuggestionPut",
+        "operationId": "customPut",
```

Comment:
I'm guessing the method name was adapted to make it more unique and avoid collision. I guess we need to to keep `customPut/customGet...` everywhere though since it's used to generate the clients right?
I don't think it's an issue, but one more reason to exclude these methods when generating tools from the specs